package.json

```diff
@@ -17,6 +16,8 @@
   "repository": "kontist/validate-steuerid",
   "author": "Kontist GmbH (kontist.com)",
   "main": "dist/index.js",
+  "module": "dist/main.js",
+  "exports": "./dist/main.js",
   "scripts": {
     "transpile": "tsc && babel --root-mode upward src --extensions \".js\" --out-dir dist --copy-files --source-maps --quiet && echo \"--ALL FILES TRANSPILED!\"",
     "test": "ts-mocha -p tsconfig.json src/**/*.spec.ts"
```

Comment:
@UgurGumushan I get the error ```TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /home/dananussair/Development/validate-steuerid/src/index.spec.ts``` when I try to run the tests.

Reply:
[yarn test should work after this commit](https://github.com/kontist/validate-steuerid/pull/3/commits/39300d0ae4e4faa371cd4b6a6d4c0615e5920c75)